examples/bots/discord-bot/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "discord-bot"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.72"
+clap = { version = "4.3.17", features = ["derive", "env"] }
+poise = "0.5.5"
+reqwest = { version = "0.11.18", features = ["json"] }
+tokio = { version = "1.29.1", features = ["rt-multi-thread", "macros"] }
```

Comment:
Don't specify patch versions unless strictly necessary.

Reply:
I assume this goes for all dependencies?